spec/v93k_dir_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe 'V93K unique test name generation' do
```

Comment:
@CodyBriscoe this needs to be renamed to reflect that this is a v93k output directory spec test